index.html

```diff
@@ -1318,7 +1318,11 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/shanna-walsh-advice.html" target="_blank">Shanna's Advice</a><br>
       <a href="advice/chris-sinclair-advice.html" target="_blank">Chris Sinclair's Advice</a><br>
       <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>
+<<<<<<< HEAD
+      <a href="advice/kassie-bartow-advice.html" target="_blank">Kassie's Advice 🦋</a><br>
+=======
       <a href="advice/april-jordan-advice.html" target="_blank">April's Advice</a><br>
+>>>>>>> main
```

Comment:
Hi @kassiebartow,

It looks like the conflict markers haven't been removed yet. If you've made this change already, check that your code has been saved and then add and commit the change and push your advice branch to GitHub. 

Let us know if you have any questions! 

Best,
Lisa